gcloud/pubsub/test_connection.py

```diff
@@ -110,54 +113,51 @@ def test_build_api_url_w_base_url_override(self):
         self.assertEqual(conn.build_api_url('/foo', api_base_url=base_url2),
                          URI)
 
-    def _verify_uri(self, uri, expected_path, **expected_qs):
```

Comment:
Correct.  We are back to working at the `connection` level, rather than the `http` level.